src/controllers/t411.js

```
@@ -25,14 +25,70 @@ let torrentRouter = () => {
 				return next(new Error(`Authentification error (${json.code}): ${json.error}`))
 			}
 			let user = db.get(req.user, {})
-			user.t411Token = json.token
+			user.t411 = {
+				token: json.token,
+				uid: json.uid
+			}
 			db.set(req.user, user)
 			res.json({})
 		})
 	})
 
+	/*
+	 * Disconnect user from t411
+	 */
+	router.delete("/auth", (req, res, next) => {
+		let user = db.get(req.user, {})
+		user.t411 = {}
+		db.set(req.user, user)
+		res.json({})
+	})
+
+	/*
+	 * Update user profile from t411 and stores result in db
+	 */
+	router.get("/profile/refresh", (req, res, next) => {
+		let token = db.get(req.user, {}).t411.token || db.get(req.user, {}).t411Token || null
+		let uid = db.get(req.user, {}).t411.uid || null
+		if (!token || !uid) {
+			return res.json(null)
```

rather than null, maybe return an error explaining that no t411 configuration was found